src/bitcoin-cli.cpp

```diff
@@ -477,7 +477,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
             const int8_t network_id{NetworkStringToId(network)};
             if (network_id == UNKNOWN_NETWORK) continue;
             const bool is_outbound{!peer["inbound"].get_bool()};
-            const bool is_block_relay{peer["relaytxes"].isNull() ? false : !peer["relaytxes"].get_bool()};
+            const bool is_tx_relay{peer["relaytxes"].isNull() ? false : !peer["relaytxes"].get_bool()};
```

Comment:
Wouldn't renaming this to `is_tx_relay` require to invert the bool? Currently, if `relaytxes==true`, `is_tx_relay==false` which seems confusing.